codegen/settingsgen.py

```diff
@@ -134,6 +134,7 @@ def _populate_hash_dict(name, info, cls, api_tree):
 
     cls_touple = (
         name,
+        cls.__bases__,
```

Comment:
just noting that the existing name above seems to be misspelled. `cls_tuple` must have been intended.

Reply:
`cls_tuple` is inside `_populate_hash_dict`.

Reply:
Be it wherever, can we not update it?

Reply:
@seanpearsonuk  I meant indentation is correct we will correct it's name.

Reply:
@seanpearsonuk Yes we will update it, I just read intended as indented so I have written that it's indented correctly.